frontend/src/components/audit-log/AuditEntryIcon.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+    <LockClosedIcon v-if="icon === 'security'" class="ff-icon" />
+    <CogIcon v-if="icon === 'settings'" class="ff-icon" />
+    <ProjectIcon v-if="icon === 'project'" class="ff-icon text-red-700" />
+    <NodeRedIcon v-if="icon === 'nodered'" class="ff-icon text-red-700" />
+    <ClockIcon v-if="icon === 'clock'" class="ff-icon text-purple-700" />
+    <DesktopComputerIcon v-if="icon === 'stacks'" class="ff-icon text-red-700" />
+    <ColorSwatchIcon v-if="icon === 'project-types'" class="ff-icon text-red-700" />
+    <ChipIcon v-if="icon === 'device'" class="ff-icon text-blue-700" />
+    <MailIcon v-if="icon === 'mail'" class="ff-icon text-green-700" />
+    <IdentificationIcon v-if="icon === 'user-profile'" class="ff-icon text-green-700" />
+    <UserIcon v-if="icon === 'user'" class="ff-icon text-green-700" />
+    <UserGroupIcon v-if="icon === 'users'" class="ff-icon text-teal-700" />
+    <CurrencyDollarIcon v-if="icon === 'billing'" class="ff-icon text-yellow-500" />
+    <KeyIcon v-if="icon === 'key'" class="ff-icon text-green-700" />
+    <LoginIcon v-if="icon === 'login'" class="ff-icon text-blue-700" />
+    <LogoutIcon v-if="icon === 'logout'" class="ff-icon text-gray-600" />
+    <ExclamationCircleIcon v-if="icon === 'error'" class="ff-icon text-red-500" />
+</template>
+
+<script>
+
```

Comment:
[nb] All the script tags in this PR have an extra blank line, is this a stylistic decision?